dictsource/ar_rules

```diff
@@ -43,11 +43,6 @@
      ۸       8
      ٩       9
      ۹       9
-     ئ       يء      // توحيد شكل الهمزة
-     ؤ       ۄء
-     إ       اءِ
-     أ       اء
-     ـ               // حذف التطويل
```

Comment:
Yes, but in positive way.
Because `.replace` rule is executed as very first, it mangles word to look in relevant entries of `ar_list` and `ar_listx` files, because there (of course) original writing is used. (I stumbled across it quite long because of #489.)